x-pack/plugins/monitoring/public/components/license/index.js

```diff
@@ -53,21 +59,29 @@ const LicenseUpdateInfoForRemote = ({ isPrimaryCluster }) => {
 export function License(props) {
   const { status, type, isExpired, expiryDate } = props;
   return (
-    <EuiPage className="licenseManagement">
+    <EuiPage>
       <EuiPageBody>
-        <div className="licManagement__contain">
-          <LicenseStatus
-            isExpired={isExpired}
-            status={status}
-            type={type}
-            expiryDate={expiryDate}
-          />
-
-          <EuiSpacer />
+        <LicenseStatus
+          isExpired={isExpired}
+          status={status}
+          type={type}
+          expiryDate={expiryDate}
+        />
+        <EuiSpacer />
 
-          <LicenseUpdateInfoForPrimary {...props} />
-          <LicenseUpdateInfoForRemote {...props} />
-        </div>
+        <LicenseUpdateInfoForPrimary {...props} />
+        <LicenseUpdateInfoForRemote {...props} />
+        <EuiSpacer />
+        <EuiText size="m">
+          <EuiTextAlign textAlign="center">
+            <p>
+              For more license options please visit&nbsp;
```

Comment:
@elastic/kibana-design Random question - do we prefer `&nbsp;` or `{' '}`? I've seen both used and I'm not sure if it matters

Reply:
Either works really, but most IDE auto-fixes to `{' '}` which is why you might see that one more often than not.